### PR TITLE
feat(claim.go): add bundleReference property

### PR DESCRIPTION
**Pull request by vdice**
* adds the `bundleReference` property to a Claim

Ref https://github.com/cnabio/cnab-spec/pull/299

### CODE REVIEW

File: claim/claim.go [hunk 1]
```diff
@@ -39,14 +39,15 @@ const (
...
-	Bundle        *bundle.Bundle              `json:"bundle"`
-	Result        Result                      `json:"result,omitempty"`
-	Parameters    map[string]interface{}      `json:"parameters,omitempty"`
+	SchemaVersion   schemaversion.SchemaVersion `json:"schemaVersion"`
+	Installation    string                      `json:"installation"`
+	Revision        string                      `json:"revision"`
+	Created         time.Time                   `json:"created"`
+	Modified        time.Time                   `json:"modified"`
+	Bundle          *bundle.Bundle              `json:"bundle"`
+	BundleReference string                      `json:"bundleReference,omitempty"`
```
Comment (radu-matei) [inline claim/claim.go]: Could we update the test for claims?

Comment (vdice) [reply claim/claim.go]: Good call; updated.

**Comment by silvin-lubecki** [comment]
It looks like the [claim schema](https://github.com/cnabio/cnab-spec/blob/master/schema/claim.schema.json) is not up-to-date 😅

**Comment by silvin-lubecki** [comment]
Ok my bad the PR is here https://github.com/cnabio/cnab-spec/pull/341 thank you @vdice :)

**Pull request merged**